resources/tax_type/ca_ab_gst.json

```diff
@@ -0,0 +1,22 @@
+{
+    "name": "Canada - Alberta",
+    "generic_label": "GST",
+    "display_inclusive": false,
```

Comment:
Considering this is the default I'd suggest just removing it from all the json files (most of the others do it this way)